std/fs/write_json_test.ts

```diff
@@ -33,14 +22,11 @@ Deno.test("writeJsonIfExists", async function (): Promise<void> {
 
   await Deno.writeFile(existsJsonFile, new Uint8Array());
 
-  await assertThrowsAsync(
-    async (): Promise<void> => {
-      await writeJson(existsJsonFile, { a: "1" });
-      throw new Error("should write success");
-    },
-    Error,
-    "should write success",
-  );
+  try {
+    await writeJson(existsJsonFile, { a: "1" });
+  } catch {
+    // empty
+  }
```

Comment:
The empty catch block seems really strange. Straightforwardly thinking, a test should either expect a throw or not, but here we acknowledge that the function might throw and ignore that fact?

Reply:
Nevermind, missed the discussion in the other comment.